cluster-operator/src/main/resources/kafkaBridgeDefaultLoggingProperties

```diff
@@ -1,9 +1,24 @@
-log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
-log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
-log4j.appender.CONSOLE.layout.ConversionPattern=%d{ISO8601} %p %m (%c) [%t]%n
-bridge.root.logger=INFO
-log4j.rootLogger=${bridge.root.logger}, CONSOLE
-log4j.logger.http.openapi.operation.healthy=WARN, CONSOLE
-log4j.additivity.http.openapi.operation.healthy=false
-log4j.logger.http.openapi.operation.ready=WARN, CONSOLE
-log4j.additivity.http.openapi.operation.ready=false
\ No newline at end of file
+name = BridgeConfig
```

Comment:
So, hang on one moment. In the bridge itself we have two copies of this config, and now in the CO we're also embedding a 3rd copy. Is that really necessary?

Reply:
This copy is from where the CO creates the configmap with the logging configuration.
The other two in the bridge project are mostly related 1) for development (in the resources) 2) in the config folder for bridge running on bare metal.

Reply:
So if the bridge classpath one is really "for development" what's it doing built into the bridge jar? If, on the other hand, that was a sensible default then there would be no need to have the default here.

Reply:
It's in the jar to avoid that you have to start your bridge in the IDE specifying a log4j2.properties file path for logging configuration.
About having it here, currently, it's placed in the configmap mounted as volume and then it is set in the bridge run script as following:

```
export KAFKA_BRIDGE_LOG4J_OPTS="-Dlog4j2.configurationFile=file:$STRIMZI_HOME/custom-config/log4j2.properties
```

using the embedded one directly then means that if we remove that line, the user cannot specify an external logging file anymore, right? And changing the default logging would mean releasing a new bridge then, while we much more release the operator often than the bridge.
Or I am just misleading your point :-)